airbyte-integrations/connectors/source-zendesk-support/source_zendesk_support/manifest.yaml

```diff
@@ -353,6 +349,32 @@ definitions:
       cursor_filter: "start_time"
       primary_key: "id"
 
+  posts_stream:
+    $ref: "#/definitions/base_incremental_stream"
+    name: posts
+    primary_key: "id"
+    incremental_sync:
+      $ref: "#/definitions/cursor_incremental_sync"
+      start_time_option:
+        $ref: "#/definitions/cursor_incremental_sync/start_time_option"
+        field_name: "start_time"
```

Comment:
tiny nit: But i think it reads a little funky to pull the existing referenced `start_time_option` to override one field and its just more readable to define a new `RequestOption` as such:
```
start_time_option:
  type: RequestOption
  inject_into: request_parameter
  field_name: "start_time

```